### ISSUE TITLE
Pwsh.Inline() Command 

### ISSUE THREAD

**Issue by erstert3st** [sharpliner/sharpliner#312]
```csharp
Pwsh.Inline("Write-Host 'iam not working'")
```

creates this yaml:

```yaml  
 - powershell: |-
        Write-Host 'iam not working'
   pwsh: true
```
It's the same problem as #168 
steps.powershell doesnt have the pwsh arg like the PowerShell@2 template.

**Comment by premun** [comment]
Hey,
Thanks for reporting, I will have a look.

**Comment by premun** [comment]
Hey @erstert3st,
This has been fixed in the version `1.5.17`.
